renovate.json

```diff
@@ -2,9 +2,6 @@
   "extends": ["config:recommended", ":pinDependencies"],
   "enabledManagers": ["npm"],
   "skipInstalls": true,
-  "postUpgradeTasks": {
-    "commands": ["yarn install"]
-  },
```

Comment:
This is causing the following errors on renovate PRs: https://github.com/bbc/simorgh/pull/11073#issuecomment-2147067305